app/models/art.rb

```diff
@@ -8,4 +8,15 @@ class Art < ApplicationRecord
   validates :title, :location, :description, :user, :photos, presence: true
   validates :price, presence: true, numericality: { greater_than: 0 }
   validates :height, :width, numericality: { only_integer: true, greater_than: 0 }, allow_nil: true
+
+  include PgSearch::Model
+  pg_search_scope :global_search,
+  against: [ :title, :description, :location ],
+  associated_against: {
+    category: :name,
+    user: %i[first_name last_name]
+  },
+  using: {
+    tsearch: { prefix: true }
+  }
```

Comment:
I haven't watched the lecture yet, my bad.
Does that mean that when you search in the form it will search among art title, art description art location, category name, user first name and user last name ?